reference-test-app/README.md

```diff
@@ -0,0 +1,79 @@
+# Table of content
+
+1. [** Description**](#description)
+
+2. [** Project's directory structure**](#projects-directory-structure)
+
+3. [** Setup guide**](#setup-guide)
+
+## Description
+
+This is a sample Roku channel utilizing the THEOplayerSDK and connectors.
+THEOplayerSDK is a custom component allowing channel developers to easily embed the THEOplayer in a channel.
+The channel presents a list of media to play using the THEOplayerSDK with the default, slightly customized Roku Video UI.
+
+## Project's directory structure
+
+- `components` contains all the components of basic playback reference application.
+- `images` directory contains the channel artworks.
+- `source` directory contains the `main.brs` file, a starting point for the Roku channel.
+- `manifest` file is required by the Roku platform, contains the configuration of the Roku channel.
+- `README.md` file is available in root folder.
+- `package.sh` optional build script to make deploying easier
+
+## Setup guide
+
+1. Install an IDE, preferably VSCode with Brightscript extensions.
+2. Connect the Roku device and turn on the developer mode: https://blog.roku.com/developer/developer-setup-guide in order to sideload the application zip onto the device.
+
+### Configuring the reference app
+
+1. Clone the reference app repository - https://github.com/THEOplayer/samples-roku-sdk.
+1. Paste the THEOplayer license in the chosen reference application files instead of the placeholder:
+   ```brightscript
+   m.player.configuration = {
```

Comment:
This looks different from the snippet in the main readme. Not a big deal though.

Reply:
Ok, I've updated it.